### PR TITLE
Feature/#17760 ch9

### PR DESCRIPTION
**Pull request by ljblancoredborder**

### CODE REVIEW

**Comment by JPeraltaNic** [comment]
@all-contributors please add @ljblancoredborder for content, translations

**Review by JPeraltaNic — CHANGES_REQUESTED** [review]
The brute force use case is using scripts from the CentOS 6 version, we need the use cases to be adapted to the Rocky 9 version of RedBorder

File: docs/guides/use_cases/images/ch30_img005_a.png
Comment (JPeraltaNic) [inline docs/guides/use_cases/images/ch30_img005_a.png:1]: `rb_set_service.sh` is from CentOS 6

**Review by JPeraltaNic — CHANGES_REQUESTED** [review]

File: docs/guides/use_cases/images/ch30_img005_a.png
Comment (ljblancoredborder) [reply docs/guides/use_cases/images/ch30_img005_a.png:1]: neither service or rbcli service is working with redborder-cep. I cannot update the images on current state.

**Review by JPeraltaNic — APPROVED** [review]

**Pull request merged**